lib/avrogen/schema/schema_registry.ex

```diff
@@ -39,10 +40,23 @@ defmodule Avrogen.Schema.SchemaRegistry do
     end)
     |> Avrogen.Schema.topological_sort()
     |> Noether.Either.map(fn schemas ->
-      json = Jason.encode!(schemas)
-      encoder = make_encoder(json)
-      decoder = make_decoder(json)
-      :ets.insert(__MODULE__, {@ets_name, json, encoder, decoder})
+      try do
+        json = Jason.encode!(schemas)
+        encoder = make_encoder(json)
+        decoder = make_decoder(json)
+        :ets.insert(__MODULE__, {@ets_name, json, encoder, decoder})
+      rescue
+        e ->
+          formatted_error = Exception.format(:error, e, __STACKTRACE__)
+
+          Logger.error("Error when attempting to make encoder/decoder.",
+            error: formatted_error,
```

Comment:
will we be able to see the logger metadata in the CI? I think we might only see the main text

Reply:
I think you get the metadata if your hipster-level is over 9000 and you click on sufficiently many links, buttons and icons, but I might be wrong.

I can add more plain text.